mycli/config.py

```diff
@@ -19,16 +21,37 @@ class CryptoError(Exception):
 
 logger = logging.getLogger(__name__)
 
-def load_config(usr_cfg, def_cfg=None):
-    cfg = ConfigObj()
-    cfg.merge(ConfigObj(def_cfg, interpolation=False))
-    cfg.merge(ConfigObj(expanduser(usr_cfg), interpolation=False))
-    cfg.filename = expanduser(usr_cfg)
+def read_config_file(f):
+    """Read a config file."""
 
-    return cfg
+    try:
+        config = ConfigObj(f, interpolation=False)
+    except ConfigObjError as e:
+        print("Error parsing line {0} of config file '{1}'.".format(
+              e.line_number, f), file=sys.stderr)
```

Comment:
Nice touch adding the line numbers. We might get some complaints regarding this error message since ConfigObj will fail on some of the constructs such as `!includedir /usr/local/etc/my.cnf.d`. But I think that is inevitable.